README.md

```diff
@@ -9,7 +12,7 @@ cd griptape-chat
 ```
 
 ## Setting up the runtime environment
-Create file called .env in the root of this project directory and add the following content:
+Copy the .env.example into a .env in the root of this project directory and add the following content:
  ```shell
  OPENAI_API_KEY=your key here
```

Comment:
Do customers need to add this content or just populate the OPENAI_API_KEY that was copied over from the .env.example?